<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVM Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QDoubleSpinBox" name="modBox">
        <property name="prefix">
         <string>Mod: </string>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="fSwBox">
        <property name="prefix">
         <string>F_SW: </string>
        </property>
        <property name="suffix">
         <string> kHz</string>
        </property>
        <property name="minimum">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="currentBox">
        <property name="prefix">
         <string>Current: </string>
        </property>
        <property name="suffix">
         <string> A</string>
        </property>
        <property name="maximum">
         <double>400.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="speedBox">
        <property name="prefix">
         <string>Speed: </string>
        </property>
        <property name="suffix">
         <string> ERPM</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>250000.000000000000000</double>
        </property>
        <property name="value">
         <double>10000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="revBox">
        <property name="prefix">
         <string>Revolutions: </string>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="tSwBox">
        <property name="prefix">
         <string>Switch Time: </string>
        </property>
        <property name="suffix">
         <string> µS</string>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="phaseLagBox">
        <property name="prefix">
         <string>Phase lag: </string>
        </property>
        <property name="suffix">
         <string> °</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>45.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="sampleV0V7Box">
        <property name="text">
         <string>Sample V0 and V7</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="currentMaxBox">
        <property name="prefix">
         <string>Max Current: </string>
        </property>
        <property name="suffix">
         <string> A</string>
        </property>
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
        <property name="value">
         <double>160.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="twoShuntBox">
        <property name="text">
         <string>Two Shunts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="highCurrentBox">
        <property name="text">
         <string>High Current Mode</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="hcVoltageCompareBox">
        <property name="text">
         <string>HC Voltage Compare</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Bus</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCustomPlot" name="plot" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="currentLabel">
            <property name="text">
             <string>I RMS:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Phases</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCustomPlot" name="plotPhase" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport_LTspice_PWL_file"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionExport_LTspice_PWL_file">
   <property name="text">
    <string>Export LTspice PWL file...</string>
   </property>
   <property name="toolTip">
    <string>Export LTspice PWL file</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
